### PR TITLE
[soltest] Print bytes range sizes

### PR DESCRIPTION
**Pull request by erak**
Closes https://github.com/ethereum/solidity/issues/6218.

### CODE REVIEW

**Comment by erak** [comment]
@ekpyron Updated :)

File: test/libsolidity/util/TestFunctionCall.cpp [hunk 1]
```diff
@@ -124,13 +124,21 @@ string TestFunctionCall::formatBytesParameters(bytes const& _bytes, dev::solidit
 	stringstream resultStream;
 	if (_bytes.empty())
 		return {};
+	auto sizeFold = [](size_t _a, Parameter const& _b) { return std::move(_a) + _b.abiType.size; };
```
Comment (ekpyron) [inline test/libsolidity/util/TestFunctionCall.cpp]:
```suggestion
	auto sizeFold = [](size_t _a, Parameter const& _b) { return _a + _b.abiType.size; };
```
No point in moving ``size_t`` :-).

**Review by ekpyron — DISMISSED** [review]
Fine apart from the minor comment!

File: test/libsolidity/util/TestFunctionCall.cpp (hunk 1 above)
Comment (erak) [reply test/libsolidity/util/TestFunctionCall.cpp]: Fxed :)

**Review by ekpyron — APPROVED** [review]

**Pull request merged**